arcgis-ios-sdk-samples/Layers/Browse OGC API feature service/BrowseOGCAPIServicesViewController.swift

```diff
@@ -0,0 +1,224 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class BrowseOGCAPIFeatureServiceViewController: UIViewController {
+    // MARK: Storyboard views
+    
+    /// The map view managed by the view controller.
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = map
+            // Set the viewpoint to Daraa, Syria.
+            mapView.setViewpoint(AGSViewpoint(latitude: 32.62, longitude: 36.10, scale: 20_000))
+        }
+    }
+    /// The bar button to browse feature layers.
+    @IBOutlet var browseBarButtonItem: UIBarButtonItem!
+    
+    // MARK: Properties
+    
+    let map = AGSMap(basemapStyle: .arcGISTopographic)
```

Comment:
Is it possible to make this a local variable? Seems like `map` is only used in `loadLayers`